dbt/node_runners.py

```diff
@@ -443,20 +443,20 @@ def print_start_line(self):
                                         self.num_nodes)
 
     def execute_test(self, test):
-        res, rows = self.adapter.execute_and_fetch(
+        res, table = self.adapter.execute_and_fetch(
             self.profile,
             test.get('wrapped_sql'),
             test.get('name'),
             auto_begin=True)
 
-        num_rows = len(rows)
-        if num_rows > 1:
-            num_cols = len(rows[0])
+        num_rows = len(table.rows)
+        if num_rows != 1:
```

Comment:
why change this?

Reply:
(specifically from `>1` to `!=1`)

Reply:
i thought i was improving the quality of this check, but maybe there's an unintended consequence here